cmd/deploy.go

```diff
@@ -1,7 +1,14 @@
 package cmd
 
 import (
+	"bufio"
+	"context"
 	"fmt"
+	"github.com/containers/image/v5/pkg/docker/config"
+	"github.com/pkg/errors"
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
Very very minor nit, I like when imports are ordered: 
1. go libs (fmt, context,..) 
2. third party libs (github.com/ory/viper, github.com/spf13/cobra) 
3. project libs (github.com/boson-project/func/knative).

I think that it is nice to follow the convetion at least for the first two points (1 and 2), currently it is quite mixed.

Reply:
I'll try run `goimports`.